Dockerfile

```
@@ -0,0 +1,61 @@
+# Copy built client assets
+COPY --from=client-build /app/client/dist /usr/share/nginx/html
+
+# Copy nginx config
+COPY client/nginx.conf /etc/nginx/http.d/default.conf
+
+# Create uploads directory
+RUN mkdir -p /app/server/uploads/avatars && \
+    chown -R appuser:appgroup /app /usr/share/nginx/html /var/log/nginx /var/lib/nginx
+
+# Start script
+RUN echo '#!/bin/sh\n\
```

Hadolint (2.14.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)